src/Microsoft.PowerShell.Commands.Utility/commands/utility/trace/SetTracerCommand.cs

```diff
@@ -29,7 +29,7 @@ public string[] Name
         }
 
         /// <summary>
-        /// The flags to be set on the TraceSource
+        /// The flags to be set on the TraceSource.
         /// </summary>
         /// <value></value>
```

Comment:
Suggest: remove the empty <value></value> comments

Reply:
Fixed.